Sector Ownership/Non-UK/Paris.txt

```diff
@@ -1,6 +1,5 @@
 SECTOR:LFFF Low:0:0
 OWNER:PN:P
-BORDER:LSFF:LFRR_LFFF:LFFF-N:LFFF_EBBU:LDFF01:LDFF02:LDM03:LDM05:LDM07:LDM08:LDM09a:LDM09b:LDM10
 
 SECTOR:LFFF Low (DB-245):0:24500
```

Comment:
I agree, my thinking was to leave it split in case theres a new FL245 split in the future, but, people can revert these simple sectors again in future if so.